src/libclipper/include/clipper/task_executor.hpp

```diff
@@ -371,6 +378,29 @@ class TaskExecutor {
             cur_model_metric.cache_hit_ratio_->increment(1, 1);
           }
         }
+
+        else {
+          if (active_containers_->get_replicas_for_model(t.model_).size() ==
```

Comment:
Nit: Instead of `else { if (condition)`, rewrite to use `else if (condition) { ...`